tests/suites/0_stateless/02_function/02_0058_function_ifnull.sql

```diff
@@ -0,0 +1,28 @@
+SET enable_planner_v2 = 1;
```

Comment:
Now plan v2 not work on cluster mode, we can comment these tests temporary.

Reply:
we can edit scripts/ci/ci-run-stateless-tests-cluster.sh to skip it

ref: https://github.com/datafuselabs/databend/pull/5958/files

Reply:
Like `./databend-test '^0[^4]_' --mode 'cluster' --run-dir 0_stateless --skip '02_0057_function_nullif'  `

Reply:
Yes. I read the python script.  It will be: `./databend-test '^0[^4]_' --mode 'cluster' --run-dir 0_stateless --skip '02_0057_function_nullif' '02_0058_function_ifnull'`